### PR TITLE
spg Chapter 2 answers

### PR DESCRIPTION
**Pull request by SimonGross**
Chapter 2 answers

### CODE REVIEW

File: chapter02/spg/question1.pl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use File::Spec;
+use Cwd;
+
+my $dir = Cwd::getcwd;
```
Comment (utopia-2) [inline chapter02/spg/question1.pl:9]: `Cwd` exports this by default, so it doesn't need a fully qualified call (unless you dislike the ambiguity of functions imported to the current namespace, in which case :+1:).

File: chapter02/spg/question2.pl [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use v5.14;
+
+use Module::CoreList;
+use Business::ISBN;
```
Comment (utopia-2) [inline chapter02/spg/question2.pl:8]: Unused import!

File: chapter02/spg/question2.pl [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+use strict;
+use warnings;
+use v5.14;
+
+use Module::CoreList;
+use Business::ISBN;
+
+foreach my $module ( sort keys $Module::CoreList::version{5.014002} ) {
+  print $module, ' ' , Module::CoreList::first_release_by_date( $module ), "\n"
+}
```
Comment (utopia-2) [inline chapter02/spg/question2.pl:12]: The question asks for the _"first release date"_ to be reported for `5.014002` modules, not the perl version they were first released in.

File: chapter02/spg/question2.pl (hunk 2 above)
Comment (SimonGross) [inline chapter02/spg/question2.pl:8]: Oops, I left it in there from when I was doing all my changes in one file.

File: chapter02/spg/question3.pl [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Business::ISBN;
+
+my $isbn = Business::ISBN->new('9781449393090');
+
+print "Printing data for ISBN 9781449393090", "\n";
```
Comment (utopia-2) [inline chapter02/spg/question3.pl:10]: It'd probably more resilient to change if the ISBN were maintained as a single variable and interpolated into this print statement. Alternatively, you can use the `Business::ISBN::as_string` method here to show the ISBN (formatted with hyphens).

**Pull request merged**